.gitignore

```diff
@@ -36,6 +36,7 @@ yarn-error.log
 /libpeerconnection.log
 testem.log
 /typings
+.nx/
```

Comment:
what is .nx?

Reply:
I guess something for building angular 18 apps? No idea.